flux/clusters/k8s-01/user-profile-api-db-exp/overlays/DEV/postgrescluster-instances.yaml

```diff
@@ -26,3 +26,16 @@
                   operator: In
                   values:
                     - hwn04.k8s-01.kontur.io
+# Switchover section
```

Comment:
@coderabbitai, in fact only switchover configuration is added, instance "hwn04" was already in place